docs/guides/qiskit-addons-mpf.ipynb

```diff
@@ -0,0 +1,117 @@
+    "\n",
+    "![Several circuits with a varying number of Trotter steps are used to compute the target observable](/images/guides/qiskit-addons/mpf-diagram.png)\n",
+    "\n",
+    "However generally the goal of simulating quantum dynamics is to measure some observable $\\mathcal{O}(t)$ which is a function of time. When using MPFs, multiple circuits -- each using $k_j$ Trotter steps -- are executed to obtain several measurements of the target observable $\\mathcal{O}_{k_j}(t)$ need to be acquired. The measurement of the target observable is then obtained by computing:\n",
+    "\n",
+    "$$ \\langle \\mathcal{O}(t) \\rangle = \\sum_j x_j(t) \\langle \\mathcal{O}_{k_j}(t) \\rangle. $$\n",
+    "\n",
+    "The essential idea is that you can reduce the overall Trotter error by approximating the time evolution operator with several product formulas with a variable number of Trotter steps instead of a single product formula. You construct a circuit for each term in the weighted sum which evolves the system according to each of the $k_j$ number of Trotter steps. Each circuit should then be executed separately on a QPU to reconstruct the results in a post-processing step. The utility of this technique can be viewed from two perspectives:\n",
+    "1. For a fixed number of Trotter steps that you are able to execute, you can obtain results with a Trotter error that is smaller in total.\n",
+    "2. For a number of Trotter steps which result in deep circuits, you can use MPF to find several shorter-depth circuits to run which results in a similar Trotter approximation error.\n",
+    "\n",
+    "\n",
+    "\n",
+    "## Determine MPF coefficients\n",
+    "\n",
+    "The core functionality of the `qiskit-addon-mpf` package lies in determining the MPF coefficients $x_j(t)$ (which may be time-dependent). Obtaining each $x_j(t)$ amounts to solving a system of linear equations $Ax=b$ where $x$ is the vector of coefficients to be determined, $A$ is a matrix which depends on each of the $k_j$ and the product formula used $S$ (i.e. the approximation order), and $b$ is a vector of constraints. This system of equations can be solved either exactly or via an approximate model which minimizes the 1-norm of the coefficients. Additionally, the choice for each $k_j$ is a heuristic process, but can be bounded by the following constraints\n",
+    "\n",
+    "1. The **largest** $k_j$ value is bounded by the highest depth of circuit that can be reliably executed\n",
+    "2. The **smallest** $k_j$ should satisfy $dt = t/k_j < 1$ since that is where the Trotter error is most well behaved\n",
+    "3. None of the coefficients $x_j$ should be close to $0$ since this would imply they don't contribute much to the MPF\n",
+    "4. Similarly, the coefficient associated with the largest $k_j$ value should not dominate, since this implies that we are just using a single product formula\n",
```

Comment:
```suggestion
    "4. Similarly, the coefficient associated with the largest $k_j$ value should not dominate, since this implies that you are using a single product formula\n",
```